### PR TITLE
Sensitivity provider parameter

### PR DESCRIPTION
**Pull request by pet-mit**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Feature

**What is the current behavior?** *(You can also link to an open issue here)*
The sensitivity parameter is configured in the itools configuration. It it thus impossible for a single application to call two different sensitivity providers, depending on the case being treated.

**What is the new behavior (if this is a feature change)?**
A new "sensitivity-provider" string parameter in RaoParameters can be used to select the sensitivity provider to be used.

### CODE REVIEW

File: loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputationImpl.java [hunk 1]
```diff
@@ -41,11 +41,13 @@
     protected ReferenceProgram referenceProgram;
     protected Network network;
     protected Map<EICode, LinearGlsk> glskMap;
+    protected String sensitivityProvider;
 
-    public LoopFlowComputationImpl(ZonalData<LinearGlsk> glsk, ReferenceProgram referenceProgram, Network network) {
+    public LoopFlowComputationImpl(ZonalData<LinearGlsk> glsk, ReferenceProgram referenceProgram, Network network, String sensitivityProvider) {
```
Comment (baptsegu) [inline loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputationImpl.java:46]: i would put the new `sensitivityProider` argument in in the `calculateLoopFlows()` method instead, the method which also takes in argument a sensitivityAnalysisParameter.

Within the RAO, we only use the `buildLoopFlowsFromReferenceFlowAndPtdf()`, which rely on a previously calculated sensitivity result, and therefore which does not run internally a sensi

File: ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/parameters/RaoParameters.java [hunk 2]
```diff
@@ -472,6 +492,7 @@ public static void load(RaoParameters parameters, PlatformConfig platformConfig)
                 parameters.setSolverSpecificParameters(config.getStringProperty("solver-specific-parameters", DEFAULT_SOLVER_SPECIFIC_PARAMETERS));
                 parameters.setPstOptimizationApproximation(config.getEnumProperty("pst-optimization-approximation", PstOptimizationApproximation.class, DEFAULT_PST_OPTIMIZATION_APPROXIMATION));
                 parameters.setForbidCostIncrease(config.getBooleanProperty("forbid-cost-increase", DEFAULT_FORBID_COST_INCREASE));
+                parameters.setSensitivityProvider(config.getStringProperty("sensitivity-provider", DEFAULT_SENSITIVITY_PROVIDER));
```
Comment (baptsegu) [inline ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/parameters/RaoParameters.java:495]: also add here the "load-flow-provider"

**Review by baptsegu — CHANGES_REQUESTED** [review]

File: ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/parameters/RaoParameters.java (hunk 2 above)
Comment (pet-mit) [reply ra-optimisation/rao-api/src/main/java/com/farao_community/farao/rao_api/parameters/RaoParameters.java:495]: fixed

File: loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputationImpl.java (hunk 1 above)
Comment (pet-mit) [reply loopflow-computation/src/main/java/com/farao_community/farao/loopflow_computation/LoopFlowComputationImpl.java:46]: makes sense, i'll change this

**Review by baptsegu — APPROVED** [review]
thanks for the changes !

**Pull request merged**